spec/view_component/storybook/stories_spec.rb

```diff
@@ -327,6 +327,62 @@
       )
     end
 
+    it "converts Stories with custom controls" do
+      expect(CustomControlStories.to_csf_params).to eq(
```

Comment:
What does CSF stand for? I've seen it all over the codebase but I haven't been able to retain the acronym.

Reply:
[Component Story Format](https://github.com/ComponentDriven/csf) Its the format Storybook stories are written in in JS/TS. This gem creates a JSON representation of that format which storybook/server turns into CSF modules.